File: University/MorattabSaziSarieS1.java
import java.util.Arrays;
import java.util.Collections;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {

            String input = scanner.nextLine();

            String[] data = input.split(" ");

            for (int i = 0; i < data.length; i++) {
                for (int j = i + 1; j < data.length; j++) {
                    String tmp;
                    if (Integer.parseInt(data[i]) > Integer.parseInt(data[j])) {
                        tmp = data[i];
                        data[i] = data[j];
                        data[j] = tmp;
                    }
                }
            }
            for (String item: data) {
                System.out.print(item + " ");
            }
        }
    }
}

File: University/MorattabSaziSarieS1.py
def main():
    input_str = input()
    data = input_str.split()

    for i in range(len(data)):
        for j in range(i + 1, len(data)):
            if int(data[i]) > int(data[j]):
                data[i], data[j] = data[j], data[i]

    for item in data:
        print(item, end=" ")

if __name__ == "__main__":
    main()
